### ISSUE TITLE
KeyError: 'inline_prop_edit'

### ISSUE THREAD

**Issue by bkuri** [IvanFon/xinput-gui#7]
I'm getting the following error as of `v0.2.3`:

```sh
Traceback (most recent call last):
  File "/usr/bin/xinput-gui", line 11, in <module>
    load_entry_point('xinput-gui==0.2.3', 'gui_scripts', 'xinput-gui')()
  File "/usr/lib/python3.7/site-packages/xinput_gui/__main__.py", line 29, in main
    settings = Settings()
  File "/usr/lib/python3.7/site-packages/xinput_gui/settings.py", line 43, in __init__
    self.load_config()
  File "/usr/lib/python3.7/site-packages/xinput_gui/settings.py", line 56, in load_config
    self.inline_prop_edit = self.config['inline_prop_edit']
KeyError: 'inline_prop_edit'
```

uname -a:
`Linux bkuri-laptop 5.1.18-1-ck-skylake #1 SMP PREEMPT Sun Jul 14 09:00:19 EDT 2019 x86_64 GNU/Linux`

python --version:
`Python 3.7.3`

**Comment by IvanFon** [comment]
This is due to the config file missing values that have been added in an update. Delete the config file (`$HOME/.xinput-gui.json`), and when you launch the program again it'll be recreated with all the necessary values.

This was fixed in master with commit 0f997967b0c21106e2cc79025d4786fe8f52dc43 and won't be a problem in future releases.

Let me know if you run into any other issues :)

**Comment by bkuri** [comment]
That did it. Thanks!
